Jenkinsfile

```diff
@@ -110,7 +110,16 @@ parallel(
 			withCredentials([[$class: 'StringBinding',
 												credentialsId: '995d99a7-466b-4beb-bf75-f3ba91cbbc18',
 												variable: 'CODECOV_TOKEN']]) {
-				sh "docker run -e CODECOV_TOKEN=\${CODECOV_TOKEN} --rm ${test_image} sh -c 'test-cover.sh && curl -s https://codecov.io/bash | bash'"
+				def codecov = "codecov -Z -C ${git_commit} "
+				if (git_branch == "remotes/origin/master") {
+					codecov += "-B master"
```

Comment:
Sorry, I should have added some more context 😄

Codecov used to use the info given by jenkins environmental variables. After switching to containers, it used git to get some information, but it didn't report on all PRs due to the SHA of GH merge commits being different. Then, I excluded the `.git` directory from the container to slim down on size (It was over 20 MB), and that entirely broke codecov.